demo/src/r2_10_6/java/com/google/android/exoplayer2/demo/SampleChooserActivity.java

```diff
@@ -231,8 +233,13 @@ private static boolean isNonNullAndChecked(@Nullable MenuItem menuItem) {
         DataSpec dataSpec = new DataSpec(Uri.parse(uri));
         InputStream inputStream = new DataSourceInputStream(dataSource, dataSpec);
         try {
-          readSampleGroups(
-              new JsonReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8)), result);
+          if (VERSION.SDK_INT >= VERSION_CODES.KITKAT) {
+            readSampleGroups(
+                new JsonReader(new InputStreamReader(inputStream, StandardCharsets.UTF_8)), result);
+          } else {
+            readSampleGroups(
+                new JsonReader(new InputStreamReader(inputStream)), result);
+          }
```

Comment:
I do not know at which point we enabled the lint checks again,, but this was causing a build to failed, so this is a quick fix, all though it is not a part of the ticket itself.